.github/workflows/chromatic.ds-react-app-launchpad.yml

```diff
@@ -1,11 +1,27 @@
 name: "react-ds-app-launchpad/chromatic"
 on:
   pull_request:
-    paths: [packages/react/ds-app-launchpad/**]
+    paths:
+      - configs/storybook/**
+      - configs/typescript-base/**
+      - configs/typescript-react/**
```

Comment:
Should updates to tsconfigs be considered grounds for re-testing? My gut says no - any regressions caused by that would likely be caught in the type-checking stage.